City-of-Helsinki/berth-reservations-admin
# VEN-251 | Edit customer info

## Description :sparkles:

* Added a new form for editing customers
* Fixed query and data problems (missing props and id's), some of which were causing Apollo errors

## Issues :bug:

### Closes :no_good_woman:

* [VEN-251](https://helsinkisolutionoffice.atlassian.net/browse/VEN-251): FE - Single customer page: edit customer info
* [VEN-626](https://helsinkisolutionoffice.atlassian.net/browse/VEN-626): Single customer page: move address data to organization model when customer group changes

## Testing :alembic:

### Automated tests :gear:️

* Added automated tests for `CustomerForm`, `CustomerEditForm` and their `utils`

### Manual testing :construction_worker_man:

* Go to the customer list
* Click a customer's name to go to the customer view page
* Click the new "Muokkaa" button on the customer profile card
* Edit values around, go nuts
* Click "Tallenna"
* The customer view page should update accordingly, without errors
